docs/Installation.md

```diff
@@ -118,19 +118,51 @@ sudo install --mode=755 ./backend_config.ini-postgresql /etc/zonemaster/backend_
 sudo install --mode=755 ./zm-centos.sh-postgresql /etc/init.d/zm-centos.sh
 mkdir "$HOME/logs"
 ```
+Get the appropriate PostgreSQL version
 
-Install, configure and start database engine (and Perl bindings):
+```sh
+sudo rpm -iUvh https://yum.postgresql.org/9.3/redhat/rhel-7-x86_64/pgdg-centos93-9.3-3.noarch.rpm
+```
 
->
-> At this time there is no instruction for using PostgreSQL on CentOS.
->
-> [Sandoche! I think we should be able to support PostgreSQL!
-> Can you try to find PostgreSQL package plus perl lib?]
->
+Install the missing dependencies:
 
-Initialize the database:
+```sh
+sudo yum -y install postgresql93 postgresql93-server postgresql93-contrib postgresql93-libs postgresql93-devel
+sudo cpan -i DBD::Pg
+```
+
+Configure:
 
 ```sh
+# Switch to root user
+sudo -s

```

Comment:
Why do you switch to root when you use sudo in the next command?

Reply:
I couldn't access the file from 'sudo'

Reply:
@sandoche2k, do you mean that you cannot do tab completion? If you have the complete path and paste that, then you will be able to edit the file through `sudo`.